bfh/fields.py

```diff
@@ -90,6 +102,13 @@ def field_name(self):
     def field_name(self, value):
         self._field_name = value
 
+    @property
+    def default(self):
+        return getattr(self, "_default", None)
```

Comment:
Do you need the guard? You are setting this attr in the init, so I don't think you'd ever encounter the AttributeError you're worried about.